wiki/Active-Effect-Guide.md

```diff
@@ -0,0 +1,440 @@
+> 
+> Source: `CONFIG.DND5E.conditionTypes`
+> </details>
+
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.traits.ci.value`     | Add     | `[conditionType]`     |
+
+#### Add a Damage Type Immunity
+
+> <details>
+> <summary>Damage Types</summary>
+> 
+> | Damage Type | Value         |
+> | ----------- | ------------- |
+> | Acid        | `acid`        |
+> | Bludgeoning | `bludgeoning` |
+> | Cold        | `cold`        |
+> | Fire        | `fire`        |
+> | Force       | `force`       |
+> | Lightning   | `lightning`   |
+> | Necrotic    | `necrotic`    |
+> | Piercing    | `piercing`    |
+> | Poison      | `poison`      |
+> | Psychic     | `psychic`     |
+> | Radiant     | `radiant`     |
+> | Slashing    | `slashing`    |
+> | Thunder     | `thunder`     |
+>   
+> Source: `CONFIG.DND5E.damageTypes`
+> </details>
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.traits.di.value`     | Add     | `[damageType]`     |
+
+#### Add a Damage Type Resistance
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.traits.dr.value`     | Add     | `[damageType]`     |
+
+#### Add a Damage Type Vulnerability
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.traits.dv.value`     | Add     | `[damageType]`     |
+
+---
+
+### Scale Value
+
+```
+system.scale.[classIdentifier].[scaleIdentifier]
+                                                .value
+                                                .number
+                                                .die
+                                                .faces
+```
+
+> <details>
+> <summary>Dice Scale Values</summary>
+> 
+> The Dice Scale Values have a few unique keys, here is an example of the result for these keys based on a scale value that is 3d8
+>
+> | Key | Value    |
+> | ------------- | -------- |
+> |`system.scale.[classIdentifier].[scaleIdentifier]`| 3d8|
+> | `system.scale.[classIdentifier].[scaleIdentifier].number` | 3 |
+> | `system.scale.[classIdentifier].[scaleIdentifier].die` | d8  |
+> | `system.scale.[classIdentifier].[scaleIdentifier].faces` | 8    |
+> </details>
+
+#### Increase the value of a Scale Value
+E.g. An Item or Spell which allows additional use(s) of a Class Feature (e.g. adds an additional use of a Barbarian's Rage).
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.scale.[classIdentifier].[scaleIdentifier].value`     | Add     | `+[number]`     |
+
+#### Increase the number of die of a Dice Scale Value
+E.g. An Item or Spell which increases the number of die in a Dice Scale Value (e.g. adds a die to a Rogue's Sneak Attack).
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.scale.[classIdentifier].[scaleIdentifier].number`     | Add     | `+[number]`     |
```

Comment:
Maybe these examples should use the concrete barbarian rage and sneak attack values in their keys. I think such an example will help a lot with understanding what is meant by 'classIdentifier' and 'scaleIdentifier'.